src/WelcomeForm/WelcomeForm.js

```diff
@@ -94,13 +101,13 @@ class WelcomeForm extends Component {
 
                     <label>Select Rank:</label>
                     <select onChange={this.handleChange.bind(this)} name="level" value={this.state.level}>
-                        <option value="select">Select a Rank</option>
-                        <option value="Padawan">Padawan</option>
-                        <option value="Knight">Knight</option>
-                        <option value="Master">Master</option>
+                        <option  value="select">Select a Rank</option>
+                        <option  value="Padawan">Padawan</option>
+                        <option  value="Knight">Knight</option>
+                        <option  value="Master">Master</option>
                     </select>
                     {errors && <span className="error"> { errors.level } </span>}
-                    <Link to='/movies' className='nav'> <button disabled={!this.state.name}> Submit I Shall </button> </Link>
+                    <Link to='/movies' className='nav'> <button onClick={this.displayUserInfo} disabled={!this.state.name}> Submit I Shall </button> </Link>
```

Comment:
This all looks good, I may change the link to a redirect once all of the information is added to the form, I will definitely let you know.